landlab/utils/source_tracking_algorithm.py

```diff
@@ -0,0 +1,189 @@
+
+"""
+26 Jan 2016
+Authors: Sai Nudurupati & Erkan Istanbulluoglu
+"""
+
+# %%
+# Import required libraries
+import numpy as np
+import copy
+from collections import Counter
+
+# %%
+def convert_arc_flow_directions_to_landlab_node_ids(grid, flow_dir_arc):
+    """
+    Convert Arc flow_directions file to be represented in node ids
+    gives receiver of each node starting at right and going clockwise.
+    
```

Comment:
Always include a one-line description on the first line of the docstring, a blank line, and then the longer description.